### PR TITLE
Lesson11 solution

### PR DESCRIPTION
**Pull request by Ognessa**

### CODE REVIEW

File: app/src/main/java/com/onix/internship/data/repository/ImageRepository.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Bitmap
+import androidx.core.graphics.drawable.toBitmap
+import com.onix.internship.R
+
+class ImageRepository(private val context: Context) {
+    private val list = mutableListOf<Bitmap>()
+
+    private val minListSize = 1
```
Comment (shadwork) [inline app/src/main/java/com/onix/internship/data/repository/ImageRepository.kt:12]: rewrite with correct collection

**Review by shadwork — CHANGES_REQUESTED** [review]
Good

File: app/src/main/java/com/onix/internship/data/repository/ImageRepository.kt (hunk 1 above)
Comment (Ognessa) [reply app/src/main/java/com/onix/internship/data/repository/ImageRepository.kt:12]: I don`t find collection which doing what I want. I find info about LinkedList and Queue but I think it don`t suitable for me.

Comment (shadwork) [reply app/src/main/java/com/onix/internship/data/repository/ImageRepository.kt:12]: take a look into https://www.bezkoder.com/kotlin-queue/ and rewrite code with Queue

Comment (shadwork) [reply app/src/main/java/com/onix/internship/data/repository/ImageRepository.kt:12]: also take a look into Stack - undo function looks like simple stack with item on tom and all others

**Pull request merged**